ios/Plugin/FilePicker.swift

```diff
@@ -21,6 +22,17 @@ import MobileCoreServices
         }
     }
 
+    @available(iOS 14.0, *)
+    public func updatedOpenDocumentPicker(multiple: Bool, documentTypes: [UTType]) {
```

Comment:
Bad naming, isn't it? (What happens if in the next iOS there is a new way to open document picker)

Maybe openDocumentPickerWithFileExtensions(...) or Similar?